kubetest/kind.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// This deployer requires --root=/go/src when pulling repositories.
+const (
+	kindBinaryPath = "/go/bin"
+	kindSrcPath    = "/go/src/sigs.k8s.io/kind"
```

Comment:
we can ask go for this instead of assuming the whole path, see:

https://github.com/kubernetes-sigs/kind/blob/c4433fe67f4daf836735dbd9aba8f38147af080b/pkg/build/kube/source.go#L24-L36